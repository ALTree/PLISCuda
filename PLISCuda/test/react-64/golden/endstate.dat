subvolumes=64
species=2
time=8.015964

0 81 
0 100 
0 89 
0 107 
0 113 
0 87 
0 103 
0 105 
0 116 
0 106 
0 85 
0 108 
0 98 
0 99 
0 99 
0 99 
0 97 
0 102 
0 104 
0 90 
0 106 
0 92 
0 95 
0 83 
0 106 
0 100 
0 109 
0 105 
0 96 
0 123 
0 93 
0 107 
0 84 
0 95 
0 97 
0 109 
0 103 
0 78 
0 105 
0 110 
0 106 
0 126 
0 96 
0 105 
0 83 
0 101 
0 107 
0 100 
0 97 
0 88 
0 93 
0 101 
0 97 
0 117 
0 104 
0 95 
0 109 
0 113 
0 94 
0 88 
0 97 
0 95 
0 101 
0 103 
