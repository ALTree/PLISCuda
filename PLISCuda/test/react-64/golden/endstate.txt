subvolumes=64
species=2
time=8.005701

0 93 
0 100 
0 110 
0 121 
0 108 
0 104 
0 103 
0 111 
0 104 
0 100 
0 98 
0 99 
0 95 
0 114 
0 109 
0 107 
0 99 
0 118 
0 113 
0 92 
0 111 
0 83 
0 108 
0 91 
0 97 
0 82 
0 84 
0 94 
0 93 
0 101 
0 66 
0 99 
0 107 
0 101 
0 118 
0 87 
0 107 
0 76 
0 109 
0 94 
0 109 
0 116 
0 104 
0 106 
0 79 
0 91 
0 95 
0 113 
0 98 
0 76 
0 83 
0 91 
0 103 
0 127 
0 101 
0 97 
0 92 
0 100 
0 105 
0 104 
0 106 
0 99 
0 88 
0 111 
